app/routes/overview/components/Overview.tsx

```diff
@@ -131,6 +143,12 @@ const Overview = () => {
           <Icon onClick={showMore} name="chevron-down-outline" size={30} />
         </div>
       )}
+
+      {frontpage.length > 8 && events.length < eventsToShow && (
+        <div className={styles.showMore}>
+          <Icon onClick={scrollToTop} name="chevron-up-outline" size={30} />
+        </div>
+      )}
     </Container>
```

Comment:
This is a good example of when to use a [turnary](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_operator), since the "condition" here is the opposite of the one above.

```ts
      {frontpage.length > 8 && events.length > eventsToShow ? (
        <div className={styles.showMore}>
          <Icon onClick={showMore} name="chevron-down-outline" size={30} />
        </div>
      ) : (
        <div className={styles.showMore}>
          <Icon onClick={scrollToTop} name="chevron-up-outline" size={30} />
        </div>
      )}
```

You can even take it a step further and say reuse the wrapper if you want

```ts
        <div className={styles.showMore}>
          {frontpage.length > 8 && events.length > eventsToShow ? (
            <Icon onClick={showMore} name="chevron-down-outline" size={30} />
           ) : (
            <Icon onClick={scrollToTop} name="chevron-up-outline" size={30} />
          )}
        </div>
```